backend/src/core/config_manager.py

```
@@ -45,7 +60,9 @@ def publish(self, event_type: str, config_data: Any) -> None:
                     callback(config_data)
                 except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)